source/core.py

```diff
@@ -424,7 +424,12 @@ def main():
 	"""
 	log.debug("Core starting")
 
-	ctypes.windll.user32.SetProcessDPIAware()
+	if winVersion.getWinVer() >= winVersion.WIN10_1703:

```

Comment:
The [documentation for `SetProcessDpiAwarenessContext`](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setprocessdpiawarenesscontext#requirements) is pretty strange with regard to the list of Windows versions it supports. It says:
> Minimum supported client
Windows 10, version 1703 [desktop apps only]
Minimum supported server
Windows Server 2016 [desktop apps only]


Which suggests that the function works under Server 2016, and the check for Windows versions above excludes it. At the same time [the documentation for `DPI_AWARENESS_CONTEXT`](https://docs.microsoft.com/en-us/windows/win32/hidpi/dpi-awareness-context) states that `DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2` is available since Windows 10 1703. Wouldn't it be safer not to rely on explicit Windows version checks, rather just try to call the function in the same order in which they're called now handling eventual `AttributeError` falling back to the older method of setting the DPI awareness if exception occurs?

Reply:
I've fixed this